src/analysis/normed_space/hahn_banach.lean

```diff
@@ -21,6 +21,32 @@ Prove more corollaries
 
 -/
 
+universes u v
+
+/-- Hahn-Banach theorem for continuous linear functions. -/
+class has_exists_extension_norm_eq (𝕜 : Type v)
+  [nondiscrete_normed_field 𝕜] [normed_algebra ℝ 𝕜] : Prop :=
```

Comment:
The assumption `[normed_algebra ℝ 𝕜]` plays a role in many applications, but is it not necessary for the definition, so I would drop it here, and only include it when needed in the applications.